apps/passport-client/src/pcdRenderers.ts

```diff
@@ -0,0 +1,53 @@
+import { EdDSAFrogPCDTypeName } from "@pcd/eddsa-frog-pcd";
+import { EdDSAFrogPCDUI } from "@pcd/eddsa-frog-pcd-ui";
+import { EdDSAPCDTypeName } from "@pcd/eddsa-pcd";
+import { EdDSATicketPCDTypeName } from "@pcd/eddsa-ticket-pcd";
+import { EdDSATicketPCDUI } from "@pcd/eddsa-ticket-pcd-ui";
+import { EmailPCDTypeName } from "@pcd/email-pcd";
+import { EmailPCDUI } from "@pcd/email-pcd-ui";
+import { EthereumOwnershipPCDTypeName } from "@pcd/ethereum-ownership-pcd";
+import { EthereumOwnershipPCDUI } from "@pcd/ethereum-ownership-pcd-ui";
+import { HaLoNoncePCDTypeName } from "@pcd/halo-nonce-pcd";
+import { HaLoNoncePCDUI } from "@pcd/halo-nonce-pcd-ui";
+import { PCDUI } from "@pcd/pcd-types";
+import { RSAPCDTypeName } from "@pcd/rsa-pcd";
+import { RSAPCDUI } from "@pcd/rsa-pcd-ui";
+import { RSATicketPCDTypeName } from "@pcd/rsa-ticket-pcd";
+import { RSATicketPCDUI } from "@pcd/rsa-ticket-pcd-ui";
+import { SemaphoreGroupPCDTypeName } from "@pcd/semaphore-group-pcd";
+import { SemaphoreGroupPCDUI } from "@pcd/semaphore-group-pcd-ui";
+import { SemaphoreSignaturePCDTypeName } from "@pcd/semaphore-signature-pcd";
+import { SemaphoreSignaturePCDUI } from "@pcd/semaphore-signature-pcd-ui";
+import { ZKEdDSAEventTicketPCDTypeName } from "@pcd/zk-eddsa-event-ticket-pcd";
+import { ZKEdDSAEventTicketPCDUI } from "@pcd/zk-eddsa-event-ticket-pcd-ui";
+
+const renderablePCDs = [
+  EdDSAPCDTypeName,
+  EdDSATicketPCDTypeName,
+  EdDSAFrogPCDTypeName,
+  EmailPCDTypeName,
+  EthereumOwnershipPCDTypeName,
+  HaLoNoncePCDTypeName,
+  RSAPCDTypeName,
+  RSATicketPCDTypeName,
+  RSATicketPCDTypeName,
+  SemaphoreGroupPCDTypeName,
+  SemaphoreSignaturePCDTypeName,
+  ZKEdDSAEventTicketPCDTypeName
+] as const;
+
+type RenderablePCDType = (typeof renderablePCDs)[number];
+
+export const pcdRenderers: { [key in RenderablePCDType]: PCDUI } = {
+  "eddsa-ticket-pcd": EdDSATicketPCDUI,
```

Comment:
FYI, this is a possible thing you could do here, I think, unless typescript is annoying:

```suggestion
  [EdDSATicketPCDTypeName]: EdDSATicketPCDUI,
```

Reply:
VSCode really wants to auto-complete the string version - but yes, if we ever changed the value of `EDDSATicketPCDTypeName` then the strings would immediately fail the type-check, which is not what we want.